src/pages/ProgramsPage/programs.jsx

```diff
@@ -0,0 +1,110 @@
+import Footer from '../../components/Footer';
+import Navbar from '../../components/Navbar';
+import People from '../../assets/people.png';
+import ProgramCarousel from './ProgramCarousel';
+import ProgramCards from './ProgramCards';
+import PersonPlaceholder from '../../assets/PersonPlaceholder.png'
+
+const ProgramSlideInfo = [
+    {title:"Mentorship",
+     description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+     link:"/programs",
+     bg: People
+    },
+    {title:"Title",
+     description:"description",
+     link:"/programs"
+    },
+];
+
+const ProgramCardInfo = [
+    {title:"MENTORSHIP",
+     description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+     link:"/programs",
+     bg: People
+    },
+    {title:"INTERNSHIP",
+     description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+     link:"/programs"
+    },
+    {title:"WORKSHOPS",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+    {title:"FUNDRAISING",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+    {title:"ALUMNI",
+     description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+     link:"/programs"
+    },
+    {title:"FAMILY",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+    {title:"MENTORSHIP",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+    {title:"FU-FIT",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+    {title:"SOCIAL",
+        description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        link:"/programs"
+    },
+];
+
+const Programs = () => {
+    return(
+    <>
+        <Navbar title="PROGRAMS" bg={People}/>
```

Comment:
There's an inconsistency with the  programs title, if you have the bandwidth try to investigate this but if anything I got it